config/metadata/partials/doi.yaml

```diff
@@ -0,0 +1,13 @@
+---
+doi:
+  type: string
+  predicate: http://purl.org/ontology/bibo/doi
+  multiple: true
```

Comment:
Why is multiple `true` here? I thought it was being set false everywhere?

Reply:
Good spot, thank you. I had forgotten to update the default YAML